core/benches/banking_stage.rs

```diff
@@ -81,9 +82,9 @@ fn bench_consume_buffered(bencher: &mut Bencher) {
                 &my_pubkey,
```

Comment:
how about perf change? Maybe, negligible?

Reply:
Heh comparing to the tip of master `f2f4003e706f7baa8534ac220c55eaa817fb1df3`, it actually seems about 10% faster, the bench went from average `92ns` to `82ns` over 10 trials. The numbers are small enough where this could be noise though

Reply:
good. cleaner and faster. It can't go wrong. xD